### PR TITLE
New package: olm-2.2.2

### PR DESCRIPTION
**Pull request by non-Jedi**
No packages currently depend on this, but I call it with some local scripts I have, and it'd be nice to see it packaged. In next release, nheko will also depend on this library.

### CODE REVIEW

File: srcpkgs/olm/template [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Template file for 'olm'
+pkgname=olm
+version=2.2.2
+revision=1
+build_style=gnu-makefile
+make_check_target="test"
+hostmakedepends="gcc python-devel python3-devel"
```
Comment (maxice8) [inline srcpkgs/olm/template]: gcc is in base-devel.

the -devel packages should be in makedepends=

File: srcpkgs/olm/template [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Template file for 'olm'
+pkgname=olm
+version=2.2.2
+revision=1
+build_style=gnu-makefile
+make_check_target="test"
```
Comment (maxice8) [inline srcpkgs/olm/template]: don't need to be quoted, it is a fixed string

File: srcpkgs/olm/template [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		vmove usr/include
+	}
+}
+
+python-olm_package() {
+  short_desc+=" - python bindings"
+	depends="${sourcepkg}>=${version}_${revision} python"
+	pycompile_module="olm"
+	pkg_install() {
+      vmove ${py2_sitelib}
```
Comment (maxice8) [inline srcpkgs/olm/template]: poor indentation

File: srcpkgs/olm/template [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+	}
+}
+
+python3-olm_package() {
+  short_desc+=" - python3 bindings"
```
Comment (maxice8) [inline srcpkgs/olm/template]: poor identation

File: srcpkgs/olm/template [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+olm-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/include
+	}
+}
+
+python-olm_package() {
```
Comment (maxice8) [inline srcpkgs/olm/template]: bindings follow <pkgname>-<language> rules.

see irssi-perl and weechat-ruby as examples.

**Comment by non-Jedi** [comment]
Thanks for review @maxice8. Comments have been resolved.

File: srcpkgs/olm/template [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+		mkdir -p ${DESTDIR}${pysl}
+		cp -r ${wrksrc}/python/olm ${DESTDIR}${pysl}
+	done
+}
+
+olm-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove usr/include
```
Comment (maxice8) [inline srcpkgs/olm/template]: add 

vmove "usr/lib/*.so"

File: srcpkgs/olm/template [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+	}
+}
+
+olm-python_package() {
+	short_desc+=" - python bindings"
```
Comment (maxice8) [inline srcpkgs/olm/template]: maybe necessary to add pycompile_modules="olm" here, not sure

is it @asergi ?

File: srcpkgs/olm/template (hunk 6 above)
Comment (non-Jedi) [reply srcpkgs/olm/template]: done

File: srcpkgs/olm/template (hunk 7 above)
Comment (non-Jedi) [reply srcpkgs/olm/template]: I have `pycompile_module=olm` a few lines down, or were you saying it might *not* be necessary?

Comment (maxice8) [reply srcpkgs/olm/template]: oh, yeah keep it, my bad.

**Comment by non-Jedi** [comment]
FYI, I've added a necessary patch I missed on my first run through to make the python bindings able to find the library.

Travis build seems to be failing for several architectures, and I can't tell why. Any tips on diagnosing?

**Comment by maxice8** [comment]
Hey, we moved to a new github organisation at https://github.com/void-linux/void-packages , can you re-open there ?

**Pull request closed without merging**